src/components/dashboard/Dashboard.js

```diff
@@ -563,6 +568,9 @@ const Dashboard = props => {
         handleFeedSelection={handleFeedSelection}
         initialNumToRender={PAGE_SIZE}
         onEndReached={nextFeed}
+        onEndReachedThreshold={0.7}
```

Comment:
add comments explaining these settings

Reply:
- I removed the 'removeClippedSubviews' prop as it is true by default - so unnecessary.
- onEndReachedThreshold prop has a clear naming.

Reply:
I asked for comments not clear naming.
I dont know what does the threshold do and dont know why you chose 0.7?